package.json

```diff
@@ -51,7 +51,8 @@
     "webpack": "4.19.1",
     "webpack-cli": "3.1.0",
     "webpack-uglify-js-plugin": "1.1.9",
-    "package-json-cleanup-loader": "1.0.3"
+    "package-json-cleanup-loader": "1.0.3",
+    "ethereumjs-util": "5.2.0"
```

Comment:
We should avoid using ethereumjs-util npm for signing. Please refer to old Master executeRule test to confirm this. For signing we should be as close to web3.